pkgs/development/libraries/science/math/clblast/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, opencl-headers
+, ocl-icd
+, OpenCL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "clblast";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "CNugteren";
+    repo = "CLBlast";
+    rev = version;
+    hash = "sha256-1FNt+lVt8AMkkEtkP61ND2ZmsMJ3ZuU2m2yIUt51mSg=";
+  };
+
+  postPatch = ''
+    substituteInPlace clblast.pc.in \
+      --replace "\''${prefix}/" "" \
+      --replace "\''${exec_prefix}/" "" \
```

Comment:
This does not look like it will work

Reply:
It does:

```
joachim@dustbowl ~/s/n/result> pwd
/home/joachim/src/nixpkgs-clblast/result
joachim@dustbowl ~/s/n/result> cat lib/pkgconfig/clblast.pc
prefix=/nix/store/c8g193qg06crwmzlnmsd8xnfmyx88bjy-clblast-1.5.3
exec_prefix=${prefix}
includedir=/nix/store/c8g193qg06crwmzlnmsd8xnfmyx88bjy-clblast-1.5.3/include
libdir=/nix/store/c8g193qg06crwmzlnmsd8xnfmyx88bjy-clblast-1.5.3/lib

Name: CLBlast
Description: CLBlast is a modern, lightweight, performant and tunable OpenCL BLAS library written in C++11
Version: 1.5.3
Libs: -L${libdir} -lclblast
Cflags: -I${includedir}
joachim@dustbowl ~/s/n/result> 
```

`exec_prefix` hast been removed from `libdir`.

Reply:
I am not sure if this is the best solution here. A patch would make this clearer.